syntax = "proto3";

import "logika/resources/enums.proto";

package logika.resources;

message L4ArchiveList
{
    repeated L4Archive list = 1;
}

message L4Archive
{
    string device = 1;
    ArchiveTypeEnum archiveType = 2;
    string channel = 3;
    string name = 4;
    string description = 5;
    DataTypeEnum recordType = 6;
    int32 recordSize = 7;
    int32 count = 8;
    int32 indexAddr = 9;
    optional int32 headersAddr = 10;
    int32 recordsAddr = 11;
    optional int32 indexAddr2 = 12;
    optional int32 headersAddr2 = 13;
    optional int32 recordsAddr2 = 14;
}
